README.md

```diff
@@ -52,6 +52,21 @@ To use a specific config, specify that with the `config-path` input:
     config-path: dprint-ci.json
 ```
 
+### Args
+
+To pass additional arguments to `dprint check`, pass them to the `args` input. E.g. to only check changed files:
+
+```yml
+- name: Get changed files
+  id: changed-files
+  uses: tj-actions/changed-files@v45
+- uses: dprint/check@v2.2
+  with:
+    args: >-
+      --allow-no-files
+      ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
```suggestion
    args: |
      --allow-no-files
      ${{ steps.changed-files.outputs.all_changed_files }}
```

I am not familiar with that yaml syntax, does a multi-line string also work?

Reply:
`>` makes it so that lines are separated with spaces instead of newlines, such that

```
a: >
  b
  c

```

Becomes

```json
{"a": "b c\n"}
```

`-` chomps the trailing newline.

Reply:
I use https://yaml-multiline.info/ ~ once per month for silly yaml features like this

Reply:
I think the problem with `|` here is that it would end up with the command

```bash
dprint check --allow-no-files
file1 file2 file3
```

Reply:
> I use [yaml-multiline.info](https://yaml-multiline.info/) ~ once per month for silly yaml features like this

It is cool and scary at the same time that yaml supports this. Thanks for explaining!